jscomp/core/initialization.ml

```diff
@@ -48,27 +53,25 @@ module Global = struct
     Clflags.strict_formats := true;
     (* Turn on [-no-alias-deps] by default -- double check *)
     Oprint.out_ident := Outcome_printer_ns.out_ident;
-    Builtin_attributes.check_bs_attributes_inclusion := Record_attributes_check.check_bs_attributes_inclusion;
+    Builtin_attributes.check_bs_attributes_inclusion :=
+      Record_attributes_check.check_bs_attributes_inclusion;
     Builtin_attributes.check_duplicated_labels :=
       Record_attributes_check.check_duplicated_labels;
     Lambda.fld_record := Record_attributes_check.fld_record;
     Lambda.fld_record_set := Record_attributes_check.fld_record_set;
     Lambda.fld_record_inline := Record_attributes_check.fld_record_inline;
-    Lambda.fld_record_inline_set := Record_attributes_check.fld_record_inline_set;
+    Lambda.fld_record_inline_set :=
+      Record_attributes_check.fld_record_inline_set;
     Lambda.fld_record_extension := Record_attributes_check.fld_record_extension;
-    Lambda.fld_record_extension_set := Record_attributes_check.fld_record_extension_set;
+    Lambda.fld_record_extension_set :=
+      Record_attributes_check.fld_record_extension_set;
     Lambda.blk_record := Record_attributes_check.blk_record;
     Lambda.blk_record_inlined := Record_attributes_check.blk_record_inlined;
     Lambda.blk_record_ext := Record_attributes_check.blk_record_ext;
     Matching.names_from_construct_pattern :=
-      Matching_polyfill.names_from_construct_pattern;
-  #ifndef BS_RELEASE_BUILD
-      Printexc.record_backtrace true;
```

Comment:
we can use `OCAMLRUNPARAM=b` if we want a stack trace

Reply:
this was a bit annoying when running tests in the dev profile.

Reply:
YES. Thank you.